vehicle/tests/conftest.py

```diff
@@ -0,0 +1,12 @@
+import pickle
+import pytest
+
+
+@pytest.fixture(scope="session")
```

Comment:
These fixtures are used by the tests. The magic strings are extracted from Redis.